### PR TITLE
ADD ISSUE_TEMPLATE

### LINKED ISSUE #22: Add an ISSUE_TEMPLATE file to this repo

### PR DESCRIPTION
**Pull request by prashant0598**
Closes #22

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/smtpapi-python?pullRequest=23) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/smtpapi-python?pullRequest=23) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/smtpapi-python?pullRequest=23) it.</sub>

**Comment by prashant0598** [comment]
@SendGridDX @thinkingserious I believe I've made all of the necessary changes. If there's something you need me to revise, let me know 😄

File: .github/ISSUE_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+1. This is the first step
+2. This is the second step
+3. Further steps, etc.
+
+Any other information you want to share that is relevant to the issue being reported. Especially, why do you consider this to be a bug? What do you expect to happen instead?
+
+#### Technical details:
+
+* sendgrid-python Version: master (latest commit: [commit number])
```
Comment (mbernier) [inline .github/ISSUE_TEMPLATE]: Please change this line to say `smtpapi-python` instead of `sendgrid-python`

Comment (prashant0598) [reply .github/ISSUE_TEMPLATE]: @mbernier now u can have a look

**Comment by thinkingserious** [comment]
Hello @prashant0598,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**